LICENSE

```diff
@@ -1,7 +1,9 @@
                     GNU AFFERO GENERAL PUBLIC LICENSE
                        Version 3, 19 November 2007
 
- Copyright (C) 2019 Maker Ecosystem Growth Holdings, INC.
+ Copyright (C) 2007 Free Software Foundation, Inc. <https://fsf.org/>
```

Comment:
This seems like an odd copyright. Is this what we should be setting it to? It breaks from our common pattern of the Dai Foundation.

Reply:
I think this is okay since it is in the license document.  I just copied it from dss: https://github.com/makerdao/dss/blob/master/COPYING. It looks like what we use in several places: https://github.com/makerdao/dss-direct-deposit/blob/master/LICENSE

Reply:
@brianmcmichael you [originally pointed this out](https://github.com/makerdao/esm/pull/24#discussion_r788037004), can you confirm this is the correct way of doing it?

Reply:
Yes, this was an error from the Foundation. They made us to change the license of the License. At the beginning we didn't know that was the case, when we figured it out, we reported it back but they took ages to give the ok to fix it and probably in several places this is still incorrect.

Reply:
and the copyright is correct now, use this one as example: https://github.com/makerdao/dss-wormhole/blob/master/LICENSE